cmd/ctr/commands/run/run_unix.go

```diff
@@ -83,6 +83,10 @@ var platformRunFlags = []cli.Flag{
 		Name:  "cni",
 		Usage: "enable cni networking for the container",
 	},
+	cli.StringSliceFlag{
+		Name:  "cdi",
+		Usage: "CDI device in the vendor.com/device=<device name> format to add to the container",
```

Comment:
nitpick: The usage string seems to imply that the accepted CDI devices names are restricted to the format mentioned. However, I don't see this being checked in the code and AFAIK the current CDI implementation does not implement such a restriction either.

Reply:
It should be done by CDI in my opinion. I'll create an issue about it.